docs/content/latest/benchmark/tpcc.md

```diff
@@ -0,0 +1,91 @@
+---
+title: TPCC
+linkTitle: TPCC
+description: TPCC
+image: /images/section_icons/architecture/concepts.png
+headcontent: Benchmark YugabyteDB using TPCC.
+menu:
+  latest:
+    identifier: benchmark-tpcc
+    parent: benchmark
+    weight: 740
+aliases:
+  - /benchmark/tpcc/
+showAsideToc: True
+isTocNested: True
+---
+
+Following are the steps to be taken to run TPCC against Yugabyte.
+
+## Step 1. Download the TPCC binaries.
+
+You can do this by running the following commands.
+
+```sh
+cd $HOME
+wget https://github.com/yugabyte/tpcc/releases/download/1.0/tpcc.tar.gz
+tar -zxvf tpcc.tar.gz
+cd tpcc
+```
+
+## Step 2. Start your database
+Start the database using steps mentioned here: https://docs.yugabyte.com/latest/quick-start/explore-ysql/.
+
+## Step 3. Configure connection properties
+Set the following connection configurations in the workload config file.
```

Comment:
It's not clear which file you are referring to:
```suggestion
Set the following connection configurations in the workload config file `config/workload_1.xml`.
```

Reply:
Done.